CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+### 2.78
+
+Release date: 2019-12-10
+
+* Fix: Resume Pipeline execution when Jenkins shutdown is canceled if the execution was automatically paused because Jenkins was preparing for shutdown. ([JENKINS-34256](https://issues.jenkins-ci.org/browse/JENKINS-34256))
```

Comment:
Maybe something like:
Fix: If a running Pipeline is paused because Jenkins is preparing for shutdown, execution will now resume if that shutdown is canceled.

Reply:
Yeah, I'm not crazy about my wording. I'll think about it a bit.

Reply:
Ok, I tried to clarify the wording in 7e85160 by splitting it into separate sentences, one simple sentence for the fix, and then a longer one explaining the difference from the previous behavior.